kornia/geometry/transform/pyramid.py

```diff
@@ -350,3 +350,49 @@ def build_pyramid(
         pyramid.append(img_down)
 
     return pyramid
+
+
+def build_laplacian_pyramid(
+    input: torch.Tensor, max_level: int, border_type: str = 'reflect', align_corners: bool = False
+) -> List[torch.Tensor]:
+    r"""Construct the Laplacian pyramid for an image.
+
+    .. image:: _static/img/build_pyramid.png
+
```

Comment:
add the citation of this paper: https://ieeexplore.ieee.org/document/1095851

check other implementations to see how we include in bibtext

```
@ARTICLE{1095851,

  author={Burt, P. and Adelson, E.},

  journal={IEEE Transactions on Communications}, 

  title={The Laplacian Pyramid as a Compact Image Code}, 

  year={1983},

  volume={31},

  number={4},

  pages={532-540},

  doi={10.1109/TCOM.1983.1095851}}

```

Reply:
For papers you can add here the bibtex and then use `:cite:` directly in the docstring